packages/tsmuxer/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/justdan96/tsMuxer
+TERMUX_PKG_DESCRIPTION="A transport stream muxer for remuxing/muxing elementary streams"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=nightly-2021-12-05-02-09-30
+TERMUX_PKG_SRCURL=https://github.com/justdan96/tsMuxer/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
Version string doesn't look good. You can use this to use date as version rather than full timestamp.
```
_VERSION_REAL=nightly-2021-12-05-02-09-30
TERMUX_PKG_VERSION=$(cut -d- -f2,3,4 <<< "$_VERSION_REAL" | tr '-' '.')
TERMUX_PKG_SRCURL=https://github.com/justdan96/tsMuxer/archive/refs/tags/${_VERSION_REAL}.tar.gz
```

Reply:
Fixed. Thanks!